src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -23,18 +15,25 @@ $animationDuration: 8s;
 
 .active::before {
   border-radius: var(--border-radius-default);
-  background-image: linear-gradient(
-    270deg,
-    var(--shade-0),
-    var(--shade-1),
-    var(--shade-2),
-    var(--shade-1),
-    var(--shade-0)
-  );
-  background-size: 400% 100%; // increase background size's width to 4x to make background animate-able
-  animation: loading $animationDuration ease-in-out infinite;
-
-  content: "";
+
+  [data-theme="system"] & {
```

Comment:
We have introduced a partial `src/styles/_theme.scss` that can be used to target dark/light modes you can use it. You can find an example inside `tajweedColors.module.scss`